wgpu/src/backend/web.rs

```diff
@@ -1263,9 +1278,22 @@ impl crate::Context for Context {
         // Can't really discard this on the Web
     }
 
-    fn device_features(&self, _device: &Self::DeviceId) -> wgt::Features {
-        // TODO
-        wgt::Features::empty()
+    fn device_features(&self, device: &Self::DeviceId) -> wgt::Features {
+        let features = device.0.features();
+
+        let features_set: js_sys::Set = features.unchecked_into();
```

Comment:
Is a runtime checked into a thing? That would solve the problem and not block us on web-sys

Reply:
Yep, https://docs.rs/wasm-bindgen/0.2.83/wasm_bindgen/trait.JsCast.html#method.dyn_into should work.

Reply:
Awesome!